content/en/developers/metrics/types.md

```diff
@@ -287,24 +289,25 @@ Submit your DISTRIBUTION type metrics from the following source:
-| ----------------- | ------------------------------------ | --------------- | -------------------- |
-| [Agent check][8]  | `self.count(...)`                    | COUNT           | COUNT                |
-| [Agent check][9]  | `self.monotonic_count(...)`          | COUNT           | COUNT                |
-| [Agent check][10] | `self.gauge(...)`                    | GAUGE           | GAUGE                |
-| [Agent check][11] | `self.histogram(...)`                | HISTOGRAM       | GAUGE, RATE          |
-| [Agent check][12] | `self.rate(...)`                     | RATE            | GAUGE                |
-| [Agent check][13] | `self.set(...)`                      | SET             | GAUGE                |
-| [API][14]         | `api.Metric.send(type="count", ...)` | COUNT           | COUNT                |
-| [API][14]         | `api.Metric.send(type="gauge", ...)` | GAUGE           | GAUGE                |
-| [API][14]         | `api.Metric.send(type="rate", ...)`  | RATE            | RATE                 |
-| [DogStatsD][15]   | `dog.gauge(...)`                     | GAUGE           | GAUGE                |
-| [DogStatsD][16]   | `dog.distribution(...)`              | DISTRIBUTION    | GAUGE, RATE          |
-| [DogStatsD][17]   | `dog.count(...)`                     | COUNT           | RATE                 |
-| [DogStatsD][17]   | `dog.increment(...)`                 | COUNT           | RATE                 |
-| [DogStatsD][17]   | `dog.decrement(...)`                 | COUNT           | RATE                 |
-| [DogStatsD][18]   | `dog.set(...)`                       | SET             | GAUGE                |
-| [DogStatsD][19]   | `dog.histogram(...)`                 | HISTOGRAM       | GAUGE, RATE          |
+| Submission Source | Submission Method (Python)                  | Submission Type | Datadog In-App Types |
+| ----------------- | --------------------------------------------| --------------- | -------------------- |
+| [Agent check][8]  | `self.count(...)`                           | COUNT           | COUNT                |
+| [Agent check][9]  | `self.monotonic_count(...)`                 | COUNT           | COUNT                |
+| [Agent check][10] | `self.gauge(...)`                           | GAUGE           | GAUGE                |
+| [Agent check][11] | `self.histogram(...)`                       | HISTOGRAM       | GAUGE, RATE          |
+| [Agent check][12] | `self.rate(...)`                            | RATE            | GAUGE                |
+| [Agent check][13] | `self.set(...)`                             | SET             | GAUGE                |
+| [API][14]         | `api.Metric.send(type="count", ...)`.       | COUNT           | COUNT                |
+| [API][14]         | `api.Metric.send(type="gauge", ...)`        | GAUGE           | GAUGE                |
+| [API][14]         | `api.Metric.send(type="distribution", ...)` | DISTRIBUTION    | GAUGE, RATE          |
```

Comment:
I think you'd want `api.Distribution.send` here. https://github.com/DataDog/datadogpy/blob/89f52f632cb97fe19d7e92e0266955fe33569afc/datadog/threadstats/reporters.py#L24

Reply:
What are the required parameters to `api.Distribution.send`?